api/apps/articles/model/save.coffee

```diff
@@ -150,6 +151,43 @@ getDescription = (article) =>
     article.sections = newSections
     cb(null, article)
 
+@indexForSearch = (article, cb) ->
+  if article.sections
+    sections = for section in article.sections
+      section.body
+
+  search.client.index(
+    index: search.index,
+    type: 'article',
+    id: article.id,
+    body:
+      slug: article.slug
+      title: article.title
+      description: article.description
+      published: article.published
+      published_at: article.published_at
+      scheduled_publish_at: article.scheduled_publish_at
+      author: article.author and article.author.name or ''
+      featured: article.featured
+      tags: article.tags
+      body: sections and stripHtmlTags(sections.join(' ')) or ''
+    , (error, response) ->
+      console.log(error) if error
+  )
+
+  cb(article)
```

Comment:
That's right I'd like the indexing to happen in the background so I'll remove the callback.